src/spatio_temporal_voxel_layer.cpp

```diff
@@ -66,65 +66,72 @@ SpatioTemporalVoxelLayer::~SpatioTemporalVoxelLayer(void)
 void SpatioTemporalVoxelLayer::onInitialize(void)
 /*****************************************************************************/
 {
-  RCLCPP_INFO(node_->get_logger(),
-    "%s being initialized as SpatioTemporalVoxelLayer!", getName().c_str());
+  // Should we initialize clock and logger here for STVL ?
+  auto node = node_.lock();
+  if (!node) {
+    throw std::runtime_error{"Failed to lock node"};
+  }
+
+  RCLCPP_INFO_STREAM(node->get_logger(), getName() <<
```

Comment:
Just for my own sanity, there's nothing technically wrong with this, can you change this and L80 back to using printf-style logging?